### PR TITLE
Feature profile

### PR DESCRIPTION
**Pull request by jesshanta**
This is for pulling data from the back-end to populate the profile data

### CODE REVIEW

**Comment by jesshanta** [comment]
screen shot 2018-03-13 at 12 29 19 am

File: ChemoTracker/components/Profile/Profile.js [hunk 1]
```diff
@@ -7,26 +7,125 @@ import Icon from 'react-native-vector-icons/MaterialIcons';
...
+			})
+			.catch(err => {
+        console.log(err);
+      })
+		})
+		.catch(err => {
+			console.log(err);
+		})
+	}
+
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/Profile.js:119]: There are some spacing issues with this file. Can you try to format the indentation please?

File: ChemoTracker/services/profileServices.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+}
+
+export const getProfile = () => {
+  return fetch(`${api}/users/profile/`, {
+    method: 'get',
+    headers: header
+  }).then(res => {return res.json()});
+};
+
+export const login = () => {
```
Comment (jpatricia) [inline ChemoTracker/services/profileServices.js:14]: just a comment, this should be changed when seun's login is merged.. so we will just need to pass in the token instead

File: ChemoTracker/components/Profile/ProfileDetailsTable.js [hunk 3]
```diff
@@ -8,86 +8,35 @@ import color from '../../styles/color.js';
...
-      datacr: [{
-        title: "Diagnosis",
-        description: "Breast Cancer",
-        inputType: "modaldropdown",
-      }, {
-        title: "Chemotherapy",
-        description: "None",
-        inputType: "text",
-      }]
+    console.log("Profile Details");
```
Comment (lsivasothy) [inline ChemoTracker/components/Profile/ProfileDetailsTable.js:11]: delete console.log

**Review by lsivasothy — COMMENTED** [review]
PR looks great! 🙂

File: ChemoTracker/components/Profile/Profile.js [hunk 4]
```diff
@@ -7,26 +7,125 @@ import Icon from 'react-native-vector-icons/MaterialIcons';
...
+		return item;
+	}
+
+	mapAllergies(allergy){
+		a = [];
+		for(var i in allergy){
+			o1 = {
+				title: "Allergen",
+				description: allergy[i].allergen,
+			}
```
Comment (jpatricia) [inline ChemoTracker/components/Profile/Profile.js:41]: ```key: `${i}_allergen` ``` might work

File: ChemoTracker/components/Profile/Profile.js (hunk 1 above)
Comment (jesshanta) [reply ChemoTracker/components/Profile/Profile.js:119]: added counter

**Review by jpatricia — APPROVED** [review]

**Pull request merged**